test_model_configs/single_population_node_with_updaters_config.txt

```diff
@@ -0,0 +1,732 @@
+                         op: EQUAL
+                         value: "18"
+                      }
+                      sub_filters {
+                         name: "email_user_info.demo.age_to"
+                         op: EQUAL
+                         value: "24"
+                      }
+                   }
+                   update {
+                      device_user_type: "email"
+                      corrected_demo {
+                        gender: 1
+                        age {
+                          min_age: 18
+                          max_age: 24
+                        }
+                      }
+                   }
+                }
+                nodes {
+                   condition {
+                      op: AND
+                      sub_filters {
+                         name: "email_user_info.demo.gender"
+                         op: EQUAL
+                         value: "1"
+                      }
+                      sub_filters {
+                         name: "email_user_info.demo.age_from"
+                         op: EQUAL
+                         value: "25"
+                      }
+                      sub_filters {
+                         name: "email_user_info.demo.age_to"
+                         op: EQUAL
+                         value: "34"
+                      }
+                   }
+                   update {
+                      device_user_type: "email"
+                      corrected_demo {
+                        gender: 1
+                        age {
+                          min_age: 25
+                          max_age: 34
+                        }
+                      }
+                   }
+                }
+                nodes {
+                   condition {
+                      op: AND
+                      sub_filters {
+                         name: "email_user_info.demo.gender"
+                         op: EQUAL
+                         value: "1"
+                      }
+                      sub_filters {
+                         name: "email_user_info.demo.age_from"
+                         op: EQUAL
+                         value: "35"
+                      }
+                      sub_filters {
+                         name: "email_user_info.demo.age_to"
+                         op: EQUAL
+                         value: "99"
+                      }
+                   }
+                   update {
+                      device_user_type: "email"
+                      corrected_demo {
+                        gender: 1
+                        age {
+                          min_age: 35
+                          max_age: 99
+                        }
+                      }
+                   }
+                }
+              }
+            }
+            updates{
+              update_matrix {
+              columns {
+                 person_country_code: "GB"
```

Comment:
This field is not set earlier in the model. Either use "labeler_input.geo.country_id" (which is an int), or set person_country_code from "labeler_input.geo.country_id" first.